java/src/jmri/jmrix/loconet/messageinterp/MessageInterpretation_nl.properties

```diff
@@ -1,7 +1,7 @@
-# Strings used by LLnmon.
+# Strings used by mesaageinterp.
```

Comment:
Looks like a misspelling. No big deal, just noticed it.

Reply:
that is my typing skill, will fix